go/ql/test/library-tests/semmle/go/dataflow/ExternalFlowInheritance/mad_i1_subtypes_true.expected

```diff
@@ -0,0 +1,40 @@
+invalidModelRow
+paths
+| test.go:8:7:8:16 | call to Source | test.go:10:9:10:9 | y |
```

Comment:
(Probably good to note these in the corresponding QL files in the manner we would with `$ MISSING` and similar if this was an inline-expectations test): I believe these are all good results except for the cases that shadow the relevant methods which are FPs.